sirepo/srunit.py

```diff
@@ -132,6 +134,10 @@ def test_in_request(op, cfg=None, before_request=None, headers=None, want_cookie
     return r
 
 
+def uwsgi_client():
+    return _UwsgiClient()
```

Comment:
Does this need to be cached like flask_client. In this case I would argue just making _UwsgiClient() public and having the test create it. It's only used in one case right now so probably don't need to cache.

Reply:
https://github.com/radiasoft/sirepo/commit/66da4701448010669a6cbfea717df20863acec62